tools/assimp_cmd/Main.cpp

```diff
@@ -325,6 +325,14 @@ bool ExportModel(const aiScene* pOut,
 		PrintHorBar();
 	}
 
+	aiMatrix4x4 rx, ry, rz;
+    aiMatrix4x4::RotationX(imp.rot.x, rx);
```

Comment:
Is this correct? Y have always used the x rotation.

Reply:
you right. my fault...
i fixed it